spec/models/user_preference_spec.rb

```diff
@@ -253,7 +253,7 @@
       it "defaults to light" do
         expect(subject.theme).to eq("light")
         expect(subject).to be_a_light_theme
-        expect(subject).to be_a_base_theme_light
+        expect(subject).to be_a_color_mode_light
```

Comment:
Would changing the preference name so that we can have `be_a_light_color_mode` be a big ordeal?

Reply:
Not at all! 👍🏾